src/integration_tests/trafficcontroller/dropsonde_end_to_end_test.go

```diff
@@ -55,6 +55,7 @@ var _ = Describe("TrafficController for dropsonde messages", func() {
 
 			var receivedEnvelope *events.Envelope
 			Eventually(messages).Should(Receive(&receivedEnvelope))
+			Eventually(errors).ShouldNot(Receive())
```

Comment:
`Eventually(...).ShouldNot(Receive())` doesn't mean anything, because it will drain the channel until the channel is empty and `Receive()` returns false.  We should be using `Consistently` here.